wremnants/datasets/datasets2016.py

```diff
@@ -18,7 +18,7 @@ def makeFilelist(paths, maxFiles=-1):
 
 def getNarfDataset(sampleName, maxFiles, sampleDict, isData, isWorZ=True):
     if isData:
-        print('Sample ', sampleName, ' read from : ', sampleDict[sampleName]["filepaths"])
+        #print('Sample ', sampleName, ' read from : ', sampleDict[sampleName]["filepaths"])
```

Comment:
You could write this with logging.debug (normally not printed out) rather than commenting it out.